### PR TITLE
Add standard SNMP host check command and host template in protocol snmp PP

### PR DESCRIPTION
**Pull request by sdepassio**
## Description

Add standard SNMP host check command and host template in applications-protocol-snmp

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 1]
```diff
@@ -78,8 +93,9 @@ Could not retrieve metrics
 
 ### Configuration SNMP
 
-Afin de récupérer des métriques via SNMP, il est nécessaire de configurer le matériel ou le système cible. Pour cela, 
-référez-vous à la documentation officielle du constructeur ou de l'éditeur.
+Afin de superviser votre ressource en SNMP,  il est nécessaire de configurer l'agent 
+sur le serveur comme indiqué sur la documentation officielle :
+* LINK
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:98]:
```suggestion
Afin de superviser votre ressource en SNMP,  il est nécessaire de configurer l'agent de l'équipement en s'appuyant sur la documentation de ce dernier.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 2]
```diff
@@ -1,38 +1,57 @@
...
 
-* App-Protocol-SNMP-custom
+Le connecteur apporte les modèles de service suivants (classés par modèle d'hôte):
+
+<Tabs groupId="sync">
+<TabItem value="App-Protocol-SNMP" label="App-Protocol-SNMP">
+
+| Alias | Modèle de service | Description |
+|:------|:------------------|:------------|
+
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:24]: Les services sont manquants.

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 3]
```diff
@@ -1,38 +1,57 @@
...
+</TabItem>
+<TabItem value="App-Protocol-SNMP-Only" label="App-Protocol-SNMP-Only">
 
-Il apporte les modèles de service suivants :
+| Alias | Modèle de service | Description |
+|:------|:------------------|:------------|
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:30]: Les services sont manquants.

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 4]
```diff
@@ -143,19 +254,19 @@ l'utilisateur **centreon-engine** (`su - centreon-engine`) :
...
 ```bash
 /usr/lib/centreon/plugins//centreon_generic_snmp.pl \
-    --plugin=apps::protocols::snmp::plugin \
-    --mode=string-value \
-    --hostname=10.0.0.1 \
-    --snmp-community='my-snmp-community' \
-    --snmp-version=2c \
-    --oid='' \
+	--plugin=apps::protocols::snmp::plugin \
+	--mode=string-value \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:258]:
```suggestion
	--mode=uptime \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 5]
```diff
@@ -143,19 +254,19 @@ l'utilisateur **centreon-engine** (`su - centreon-engine`) :
...
+	--hostname=10.0.0.1 \
+	--snmp-community='my-snmp-community' \
+	--snmp-version=2c  \
+	--oid='' \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:262]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 6]
```diff
@@ -143,19 +254,19 @@ l'utilisateur **centreon-engine** (`su - centreon-engine`) :
...
     --format-ok='' \
     --format-warning='' \
     --format-critical='' \
     --format-unknown='' \
     --warning-regexp= \
     --critical-regexp= \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:268]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 7]
```diff
@@ -164,26 +275,153 @@ La commande devrait retourner un message de sortie similaire à :
 OK: |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:275]:
```suggestion
OK: System uptime is: 3m 25s | 'uptime'=205s;;;0;
```

Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:275]:
```suggestion
OK: System uptime is: 3m 25s | 'uptime'=205s;;;0;
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 8]
```diff
@@ -78,8 +93,9 @@ Could not retrieve metrics
 
 ### Configuration SNMP
 
-Afin de récupérer des métriques via SNMP, il est nécessaire de configurer le matériel ou le système cible. Pour cela, 
-référez-vous à la documentation officielle du constructeur ou de l'éditeur.
+Afin de superviser votre ressource en SNMP,  il est nécessaire de configurer l'agent 
+sur le serveur comme indiqué sur la documentation officielle :
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:97]: à compléter

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md (hunk 1 above)
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:98]: à compléter

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 9]
```diff
@@ -123,17 +192,59 @@ yum install centreon-pack-applications-protocol-snmp
...
+</Tabs>
+
+
+### Service 
+
+Une fois le modèle appliqué, les macros ci-dessous indiquées comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Generic-Value" label="Generic-Value">
+
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:225]: Tableau à revoir

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by omercier — CHANGES_REQUESTED** [review]
I change the FR version only

**Review by omercier — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**